src/systemctl/systemctl-show.c

```diff
@@ -821,18 +821,37 @@ static void print_status_info(
 }
 
 static void show_unit_help(UnitStatusInfo *i) {
+        bool previous_man_page = false;
+
         assert(i);
 
         if (!i->documentation) {
                 log_info("Documentation for %s not known.", i->id);
                 return;
         }
 
-        STRV_FOREACH(p, i->documentation)
-                if (startswith(*p, "man:"))
-                        show_man_page(*p + 4, false);
-                else
-                        log_info("Can't show: %s", *p);
+        STRV_FOREACH(doc, i->documentation) {
+                const char *p;
+                _cleanup_free_ char *t = NULL;
+
+                p = startswith(*doc, "man:");
+
+                if (p ? doc != i->documentation : previous_man_page) {
+                        puts("");
+                        fflush(stdout);
+                }
+
+                previous_man_page = p;
+
+                if (p)
+                        show_man_page(p, /* null_stdio= */ false);
+                else {
+                        if ((p = startswith(*doc, "file://")))
```

Comment:
As per https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda#file-uris-and-the-hostname these urls should probably be rewritten so that: `file:///foo/bar` becomes `file://gethostname()/foo/bar`, for compat with ssh remoting.

Also, I think we should also urlify http/https links here (but of course not rewrite them)

Reply:
Update to use `terminal_urlify_path()` which inserts the hostname.

I didn't add urlification of http/https links: this would additional code, which doesn't seem necessary. Terminal emulators handle http/https/ftp and similar links on their own, so if we don't do anything, the user gets the correct behaviour anyway. I adjusted the commit message to mention this.